docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudwatch-firehose.asciidoc

```diff
@@ -0,0 +1,270 @@
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter]]
+=== Create a subscription filter for Amazon Data Firehose 
+
+- Choose destination
+- Grant permission
+- Configure log format and filters
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter-destination]]
+==== Choose destination
+
+Please select the Firehose stream we create in the previous step.
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter-permission]]
+==== Grant permission
+
+Grant the CloudWatch service to send log events to the stream in Firehose.
+
+This step consists of two parts::
+
+1. Create a new role with a trust policy that allows CloudWatch service to assume the role.
+2. Assign a policy to the role that permits "putting records" into a Firehose  stream.
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter-permission-role]]
+===== Create a new role
+
+Create a new IAM role and use the following JSON as the trust policy:
+
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "logs.<REGION>.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringLike": {
+                    "aws:SourceArn": "arn:aws:logs:<REGION>:<ACCOUNT_ID>:*"
+                }
+            }
+        }
+    ]
+}
+----
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter-permission-policy]]
+===== Assign a policy to the IAM role
+
+Using the the following JSON, create a new IAM policy and assign it to the role:
+
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "firehose:PutRecord",
+            "Resource": "arn:aws:firehose:<REGION>:<ACCOUNT_ID>:deliverystream/<YOUR_FIREHOSE_STREAM>"
+        }
+    ]
+}
+----
+
+When the new role is ready, you can select it in the subscription filter.
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter-log-format]]
+==== Configure log format and filters
+
+Select the "Other" in the **Log format** option.
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter-log-format-more]]
+===== More on log format and filters
+
+You can use the *Subscription filter pattern* in the subscription filter to forward only the log events that match the pattern. You can test filter patterns using *Test pattern* in the AWS console.
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-additional-logs]]
+==== Generate additional logs
+
+Open the AWS Lambda page again, select the function we created, and execute it a few times to generate new log events.
+
+[discrete]
+[[firehose-cloudwatch-step-verify]]
+=== Verify if there are destination errors
+
+Check if there are destination error logs.
+
+On the AWS console, visit your Firehose stream and check for entries in the "Destination error logs" section.
```

Comment:
I'm not looking at the UI, but if this is a labelled section in the UI, you probably want bold instead of quotes here.
```suggestion
On the AWS console, visit your Firehose stream and check for entries in the *Destination error logs* section.
```